src/sources/file.rs

```diff
@@ -142,6 +148,7 @@ pub struct FileConfig {
     ///
     /// Off by default, the offset is only added to the event if this is set.
     #[serde(default)]
+    #[configurable(metadata(docs::examples = "offset"))]
     pub offset_key: Option<OptionalValuePath>,
 
     /// Delay between file discovery calls, in milliseconds.
```

Comment:
remove reference to milliseconds?

Reply:
Resolved in [192c429](https://github.com/vectordotdev/vector/pull/16099/commits/192c429cac97c0da46a361108a4d292eeef97f2a)